.gitattributes

```diff
@@ -0,0 +1 @@
+*.sh text eol=lf
```

Comment:
Is this preserving linux newlines?

Reply:
Just as a warning here, we haven't checked for windows compatibility for much of `spin`'s functionality. The likely set of gotchas is probably present (e.g. file paths).

Reply:
Yes, that ensures that Bash scripts keep Linux newlines even when checked out on a Windows machine. By default, text files get Windows newlines when checked out on Windows, which Bash doesn't like if you're using WSL to run Linux on Windows.

Also, I've confirmed basic functionality in Windows, though I haven't tested anything involving file paths yet.